backend/src/types/context.ts

```diff
@@ -1,2 +1,8 @@
+import { InMemoryDatasource } from '../database/InMemoryDatasource';
+
 export interface Context {
+    userId: string;
+    dataSources: {
+        databaseAPI: InMemoryDatasource
+    };
 }
```

Comment:
NO ⭐ For a successful rebase on homework/main
⭐ For not having security issues according to instructions.
⭐ For requesting a review and reviewing another team's PR according to the instructions.